src/features/messages/pi-queries/fetchPiChainMessages.ts

```diff
@@ -127,23 +127,23 @@ async function fetchLogsForAddress(
       // [processTopic0, null, null, dispatchTopic],
     ],
     mailbox,
-    chainId,
+    domainId!,
     query,
     multiProvider,
   );
 }
 
 async function fetchLogsForTxHash(
-  { chainId }: ChainMetadata,
+  { domainId }: ChainMetadata,
   query: PiMessageQuery,
   multiProvider: MultiProvider,
 ): Promise<ExtendedLog[]> {
   const txHash = query.input;
-  logger.debug(`Fetching logs for txHash ${txHash} on chain ${chainId}`);
-  const provider = multiProvider.getProvider(chainId);
+  logger.debug(`Fetching logs for txHash ${txHash} on chain ${domainId}`);
```

Comment:
since we have the `ChainMetadata` object here anyway, I wonder if we should log the chain name instead. Would be more helpful for debugging. I would have done that in the first place but, like the warp ui, the first version of this app used chainId everywhere because that's what the broader standard is

Reply:
I think `name (domain)` would work well